### DISCUSSION TITLE
feature request: support for hidden accounts (or at least proper export/import)

### DISCUSSION THREAD

**Discussion by colemickens** [LedgerHQ/ledger-live-desktop#3669] [Ideas]
So far, it doesn't seem like Ledger Live has any sort of multi-account support. If I unlock my Ledger with an additional passphrase then Ledger Live gets very confused.

Trezor Suite accepts the password on the computer-side, which feels less secure, but the overall UX around hidden accounts feels much more usable on Trezor Suite.

Similarly, it might be enough if it were simply possible to export/import accounts (aka, the types, the labels, the gaps, etc). But this also seems like an intentionally not-implemented feature maybe?

**Comment by fti7** [comment]
+1
I would like to see a similiar Implementation as e.g. the KeePass Password Manager is working today.

Ledger Live itself does not store any User Data in its folders. Instead you can open a external Database File (Basically the SQLITE DB). 
Call it Ledger Live Tresor/Vault/Database or something.

Each Database has its own Password which will be used to Decrypt it. Everything from the User is then stored in there. Accounts, Settings etc.

So Users can easily switch between multiple User Databases (e.g. Different Persons) and/or also between the Main and the Hidden Account.
